pkg/pillar/types/volumes.go

```diff
@@ -131,7 +131,7 @@ func (config VolumeConfig) LogCreate() {
 		return
 	}
 	logObject.CloneAndAddField("origin", config.Origin).
-		AddField("target-size-bytes", config.TargetSizeBytes).
+		AddField("max-vol-size", config.MaxVolSize).
```

Comment:
This new field should be named max-vol-size-int64 based on the logging requirements in docs/

Reply:
Updated.